src/mind_agents/core/display.py

```diff
@@ -1,13 +1,13 @@
 """Display utilities for The Mind game."""
 
-import math
-from typing import Optional
-
-from core.game import GameState, GameStateInfo, PlayerAction
 from rich.console import Console
 from rich.panel import Panel
 from rich.table import Table
 
+from mind_agents.core.game import GameState, GameStateInfo, PlayerAction
```

Comment:
Usually bar imports come first, and all imports are alphabetical

Reply:
this commit organizes the import, and I noticed some imports not being used, a reshuffle in that top part:
223a0c36d5f49c763389feefa6177f230c0bc8c2